spec/models/user_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 describe User do
 
-  let(:therapist) { FactoryGirl.create(:therapist_user) }
-  let(:donor) { FactoryGirl.create(:donor_user) }
+  let(:therapist) { FactoryGirl.build(:therapist_user) }
```

Comment:
We should be committing the records to the db in case we do need to use the IDs, though - our system runs tests in a transaction so any created records are rolled back at the end.